_posts/2016-03-05-Content.md

```diff
@@ -5,6 +5,7 @@ menutitle:         "Content styles"
 category:          Features
 author:            jwillmer
 tags:              Jekyll Content Demo
+comments:          true
```

Comment:
Please remove the comment from this post and add it instead to `2016-04-06-YAML-Features.md`. Also please add a short description about this feature in the `2016-04-06-YAML-Features.md` post.